awscli/topics/config-vars.rst

```diff
@@ -476,6 +478,66 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+The AWS CLI uses a pager for output data that does not fit on the screen.
+
+On Linux/MacOS, ``less`` is used as the default pager. On Windows,
+the default is ``more``.
+
+Configuring pager
+^^^^^^^^^^^^^^^^^^
+
+You can override the default pager with the following configuration
+options. These are in order of precedence:
+
+    ``AWS_PAGER`` environment variable
+
+    ``cli_pager`` shared config variable
+
+    ``PAGER`` environment variable
```

Comment:
Can we use bullet points here for the listing precedence instead of a block quote? So:
```
* ``AWS_PAGER`` environment variable
* ``cli_pager`` shared config variable
* ``PAGER`` environment variable
```
The format looks weird as a block quote otherwise